packages/cli/src/commands/env.ts

```
@@ -418,75 +437,288 @@
+        break;
+
+      case 'globalDb':
+        if (usingExternalPostgres) {
+          actions.warning.push(
+            'External PostgreSQL database detected. Database data cannot be reset but local database cache files will be removed.'
+          );
+        }
+
+        let anyGlobalDbDeleted = false;
+
+        // Try deleting db dir
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)